core/src/main/codegen/config.fmpp

```diff
@@ -487,6 +487,7 @@ data: {
     includeTopN: false
     includeInsertWithOmittedValues: false
     includeHostVariables: false
+    includeExceptExpression: false
```

Comment:
nit: While you're here, can you rearrange these constants so that they're in alphabetical order (same for other config.fmpp files)?

Reply:
Thanks for doing this!